src/js/components/Box.js

```diff
@@ -62,6 +62,7 @@ export default class Box extends Component {
     this._addPropertyClass(classes, CLASS_ROOT, 'separator');
     this._addPropertyClass(classes, CLASS_ROOT, 'size');
     this._addPropertyClass(classes, CLASS_ROOT, 'textAlign', 'text-align');
+    this._addPropertyClass(classes, CLASS_ROOT, 'width');
```

Comment:
Looks like it's only setting it to a fixed width.

```
$size-xsmall: round($inuit-base-spacing-unit * 20) !default; // 480px
$size-small: round($inuit-base-spacing-unit * 24) !default; // 576px
$size-medium: round($inuit-base-spacing-unit * 30) !default; // 720px
$size-large: round($inuit-base-spacing-unit * 40) !default; // 960px
```

I can add to the size attr instead. I think by allowing the user to choose between `auto`, `inherit`, and `intitial` it gives a lot for flexibility for nesting of the <Box/> component, and will allow for better responsiveness.